consumers/hidden-test-consumer/src/hidden_test_consumer/aws_client.py

```
@@ -0,0 +1,36 @@
+from dotenv import load_dotenv
+
+load_dotenv()
+import os
```

Ruff (0.8.2)

Module level import not at top of file

(E402)